_data/chains/eip155-2049.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "Movo Smart Chain Mainnet",
+  "chain": "MOVO",
+  "rpc": [
+    "https://msc-rpc.movoscan.com",
+    "https://msc-rpc.movochain.org",
+    "https://msc-rpc.movoswap.com"
+  ],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "Movo Smart Chain",
+    "symbol": "MOVO",
+    "decimals": 18
+  },
+  "infoURL": "https://movo.uk",
+  "shortName": "movo",
+  "chainId": 2049,
+  "networkId": 2049,
+  "slip44": 2050,
+  "icon": "movo",
+  "explorers": [
+    {
+      "name": "movoscan",
+      "url": "https://movoscan.com",
+      "icon": "movoscan",
+      "standard": "EIP3091"
```

does not seem to be EIP3091 compatible
```suggestion
      "standard": "none"
```

---

Thank you!!

---

I have updated it again

---

but u did it wrong - not nonce - but "none" - would have been easier if you just committed my suggestion

---

Sorry, i updated